### PR TITLE
[RCL-183] vignette updates

### PR DESCRIPTION
**Pull request by patr1ckm**
This is a quick vignette update that includes stacking, MLP, and hyperband.

We update the online docs separately.

### CODE REVIEW

**Comment by patr1ckm** [comment]
Cool. Let's just let this one hang until external release.

**Comment by elsander** [comment]
Ok, this should be good. I have no idea how we should handle reviewing for this PR, since I was reviewing but I made the most recent changes... @patr1ckm, can you take a look at this, and I'll wait to give an approve review until you approve it?

File: vignettes/civis_ml.Rmd [hunk 1]
```diff
@@ -77,11 +77,13 @@ m <- civis_ml_random_forest_classifier(tab,
       excluded_columns = "residential_zip")
 ```
 
+Note that if the dependent variables have null values, those rows will be removed before modeling.
+
 ### Hyperparameter Tuning
 
-You can tune hyperparamters using one of two methods: grid search or hyperband. CivisML will perform grid search if you pass a named list of hyperparameters and candidate values to `cross_validation_parameters`. You can run hyperparameter tuning in parallel by setting the `n_jobs` to however many jobs you would like to run in parallel.
+You can tune hyperparamters using one of two methods: grid search or hyperband. CivisML will perform grid search if you pass a named list of hyperparameters and candidate values to `cross_validation_parameters`. By default, hyperparameter tuning will run in parallel, using as many jobs as possible without overloading your computing cluster. If you wish to have more control over the number of jobs running at once, you can set it using the `n_jobs` parameter.
```
Comment (patr1ckm) [inline vignettes/civis_ml.Rmd]: `hyperparmters` -> `hyperparameters`

File: vignettes/civis_ml.Rmd [hunk 2]
```diff
@@ -108,6 +110,24 @@ m_grid <- civis_ml(tab, dependent_variable = "upgrade",
...
+classifier and regressor was chosen to skew the distribution toward
+small values, ranging between .0003 and .03, with a mean close to
+.006. Similarly, the truncated exponential distribution for the random
+forest and extra trees models skews toward small values, ranging
+between .01 and 1, and with a mean close to .1.
+
 ### Stacking 
 
 The `"stacking_classifier"` model stacks together the `"sparse_logistic"`, `"gradient_boosting_classifier"`, and `"random_forest_classifier"` models using defaults documented in `?civis_ml`. Each row is first [standardized](http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.Normalizer.html), and then the model predictions are combined using [LogisticRegressionCV](http://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegressionCV.html) with `penalty='l2'` and `tol=1e-08`. The `"stacking_regressor"` works similarly, stacking together the `"sparse_linear_regressor"`, `"gradient_boosting_regressor"`, and `"random_forest_regressor"` models, and combining them using [NonNegativeLinearRegression](https://github.com/civisanalytics/civisml-extensions).
```
Comment (patr1ckm) [inline vignettes/civis_ml.Rmd]: `Each row` -> `each column`

File: vignettes/civis_ml.Rmd [hunk 3]
```diff
@@ -19,7 +19,7 @@ While `civis_ml` is a complex function with many arguments, basic machine learni
```
Comment (patr1ckm) [inline vignettes/civis_ml.Rmd]: `fictious` -> `ficticious`

**Comment by patr1ckm** [comment]
Also, if it's easier to just scrap the PR and make a new one, that's fine too

**Comment by elsander** [comment]
I had originally been planning to make a new PR, but I apparently set up my branch to push to this PR, and then it wouldn't let me select the branch to create a new PR (shrug). Anyway, I think this is fine, we should just both approve before merging.

**Comment by elsander** [comment]
Why rebase? This should be up to date. My plan was to squash into one commit and merge.

**Comment by patr1ckm** [comment]
This works for me!

**Comment by elsander** [comment]
Ok, all of the merge conflicts and diff weirdness should be gone now. Can you take another look @patr1ckm?

**Comment by patr1ckm** [comment]
LGTM!
